website/docs/r/hci_cluster.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Azure Stack HCI"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_hci_cluster"
+description: |-
+  Manages an Azure Stack HCI Cluster.
+---
+
+# azurerm_hci_cluster
+
+Manages an Azure Stack HCI Cluster.
+
+## Example Usage
+
+```hcl
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_hci_cluster" "example" {
+  name                = "example-cluster"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  client_id           = data.azurerm_client_config.current.client_id
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+
+  tags = {
+    ENV = "Prod"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this HCI Cluster. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the HCI Cluster should exist. Changing this forces a new resource to be created.
+
+* `location` - (Required) The Azure Region where the HCI Cluster should exist. Changing this forces a new resource to be created.
+
+* `client_id` - (Required) The Client ID of the Azure Active Directory which is used by the HCI Cluster. Changing this forces a new resource to be created.
+
+* `tenant_id` - (Required) The Tenant ID of the Azure Active Directory which is used by the HCI Cluster. Changing this forces a new resource to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the HCI Cluster.
```

Comment:
for all of these, can we use `Azure Stack HCI Cluster` rather than `HCI Cluster`

Reply:
updated